slackEvents.php

```diff
@@ -1,97 +1,54 @@
 <?php
 
 class SlackEvents {
+
+    //const REGEX_LEVEL='/^E([\d]*)$/m';
+    //const REGEX_NUMBER_ATTENNDEE='/^([\d]*)P$/m';
+    const LEVEL_LUT = [
```

Comment:
just to make sure I have a good understanding of this code: what does "LUT" stands for?

Reply:
LUT stands for "Lookup table" :)